src/SDKs/SqlManagement/Sql.Tests/Utilities/TestEnvironmentUtilities.cs

```diff
@@ -9,6 +9,75 @@ namespace Sql.Tests
 {
     public static class TestEnvironmentUtilities
     {
+        private const string _environmentVariableName = "TEST_CSM_ORGID_AUTHENTICATION";
+
+        private static readonly TestEnvironment _environment = 
+            new TestEnvironment(Environment.GetEnvironmentVariable(_environmentVariableName));
+
+        // We now load default locations from environment variable.
+        // Tests were recorded with this appended to DefaultLocationId=japaneast;DefaultLocation=Japan East;DefaultSecondaryLocationId=centralus;DefaultSecondaryLocation=Central US;DefaultStagePrimaryLocation=North Europe;DefaultStageSecondaryLocation=SouthEast Asia;DefaultEuapPrimaryLocation=East US 2 EUAP;DefaultEuapPrimaryLocationId=eastus2euap
+        public static string DefaultLocation
+        {
+            get
+            {
+                return GetValueFromEnvironment("DefaultLocation");
```

Comment:
@nathannfan this will make users that take dependency on TestEnvironmentUtilities to know these new key-value pairs that needs to be supplied in ConnectionString?
When the earlier comment was made of not hard-coding regions in tests, it does not mean you hard-code it from connection string. This will simply allow you to do the same thing from Connection string.
Consider:
1) Have a function that randomly selects location for a scenario.
2) This allows you to record each test on a new supported regions (on every record)